### DISCUSSION TITLE
Emulation mode to prevent slow down

### DISCUSSION THREAD

**Discussion by jadasse** [86Box/86Box#1514] [Ideas]
What if there was an emulation mode for machines that tend to slow down, for instance, 56%, on PCs that aren't that powerful?

**Comment by bachimusprime** [comment]
We would have to completely rewrite the emulator's timing system to do that, which a) just isn't feasible, and b) against the goals of the emulator in the first place.

**Reply by jadasse** [reply]
Oh ok
